### DISCUSSION TITLE
Filter date

### DISCUSSION THREAD

**Discussion by Sasaa22222** [microsoft-search/pnp-modern-search#1923] [Q&A]
For the column "Date" i have used RefinableDateXX

1. the filter values from my date are displayed incorrectly (The date displayed in the filter is different from the one in the column.
) Screenshot: 1. And i want also only the Date
<img width="115" alt="1" src="https://user-images.githubusercontent.com/73344482/157626432-0e500e26-75c2-4e1f-b63b-603292769609.png">

2. How can I display the documents that have the value "Unlimited" in the Date column? (See screenshots)

<img width="113" alt="2" src="https://user-images.githubusercontent.com/73344482/157625787-343cf43c-948d-4d3c-bfd0-58a1773f330e.png">
<img width="181" alt="3" src="https://user-images.githubusercontent.com/73344482/157625789-9e3e2d53-d2b5-497c-9f16-370a404da3fa.png">

Thank you

**Comment by wobba** [comment]
Can you rephrase the question to make it clearer? If you want all results, then don't add RefinableDate to your query. If you want to aggregate filters on a day, then you need to either use the static date filter layout, or create something yourself to make this work.
